api4/apitestlib.go

```diff
@@ -212,6 +212,8 @@ func (me *TestHelper) InitBasic() *TestHelper {
 	me.LoginBasic()
 	me.Group = me.CreateGroup()
 
+	me.App.Config().ServiceSettings.CreateBotAccounts = model.NewBool(true)
```

Comment:
I don't think this is something we should configure at this level -- if tests need it set one way or another, let's make it explicit in each affected test.

Reply:
Yes this makes sense. Also, undoing this change will allow for proper unit tests.